setup.py

```diff
@@ -258,6 +258,21 @@ def build_extensions(self) -> None:
             ]
             subprocess.check_call(install_args, cwd=self.build_temp)
 
+    def run(self):
+        # First, run the standard build_ext command to compile the extensions
+        super().run()
+
+        # copy vllm/vllm_flash_attn/*.py from self.build_lib to current
+        # directory so that they can be included in the editable build
+        import glob
+        files = glob.glob(
+            os.path.join(self.build_lib, "vllm", "vllm_flash_attn", "*.py"))
+        for file in files:
+            dst_file = os.path.join("vllm/vllm_flash_attn",
+                                    os.path.basename(file))
+            print(f"Copying {file} to {dst_file}")
+            self.copy_file(file, dst_file)
```

Comment:
Why is this necessary? Is the install step for ``vllm_flash_attn_c` not enough?

Reply:
see https://github.com/vllm-project/vllm/pull/8699#issuecomment-2365483678